pymc3/distributions/discrete.py

```diff
@@ -394,6 +394,10 @@ def logcdf(value, p):
             p <= 1,
         )
 
+    def get_moment(value, size, p):
+        p = at.full(size, p)
+        return at.switch(p < 0.5, np.int64(0), 1)
```

Comment:
```suggestion
        return at.switch(p < 0.5, np.int64(0), np.int64(1))
```

Reply:
https://github.com/pymc-devs/pymc3/pull/5025

Reply:
Why not use the version already in that PR?

Reply:
I rebased onto `main` which now includes those changes already.